lib/OpenQA/Utils.pm

```diff
@@ -7,6 +7,7 @@ use IPC::Run();
 use Mojo::URL;
 use Regexp::Common 'URI';
 use Try::Tiny;
+use POSIX qw(ceil);
```

Comment:
Yeah, i'm dropping that out... we can stick to using interval as an integer and that's it.